vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -92,12 +93,21 @@ const (
 	DockerUSocket           = "unix:///var/run/docker.sock"
 	defaultSleepIntervalSec = 1
 	dockerConnTimeoutSec    = 2
+	refCountDelayMin        = 10
```

Comment:
To be consistent with rest of variables, I would add postfix of Sec e.g. refCountDelayMinSec, refCountDelayMaxSec.

Why 10-20Sec. Shouldn't we be more agressive 1-10secs?